User_Story#1/SortingApp/ViewModel/CollectionSortingViewModel.cs

```diff
@@ -202,7 +203,12 @@ private void SaveArray(object obj)
         /// <owner>Anton Petrenko</owner>
         private void ShowDialogWindow()
         {
-            using (var openFileDialog = new OpenFileDialog())
+            using (var openFileDialog = new OpenFileDialog()
+            {
+                FileName = "Select a text file",
+                Filter = "Text files (*.txt)|*.txt",
+                Title = "Open text file"
+            })
```

Comment:
Please look at all properties with setter and make sure you have a check if value is equal to the cached value in field.
It is very important on VM side to avoid notifying UI without real changes.